<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_trem1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>110</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: purple}</string>
    </property>
    <property name="text">
     <string>T1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho_vertical_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>110</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho_vertical_1">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>21</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho_vertical_3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>110</y>
      <width>21</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho_horizontal_1">
    <property name="geometry">
     <rect>
      <x>259</x>
      <y>110</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho_horizontal_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>230</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>160</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: orange}</string>
    </property>
    <property name="text">
     <string>T2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho_vertical_6">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>230</y>
      <width>21</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho_vertical_5">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>230</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho_vertical_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho_horizontal_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho_vertical_7">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>230</y>
      <width>21</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>290</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: rgb(114, 159, 207)}</string>
    </property>
    <property name="text">
     <string>T3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>350</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: red}</string>
    </property>
    <property name="text">
     <string>T4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem5">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>290</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: rgb(138, 226, 52)}</string>
    </property>
    <property name="text">
     <string>T5</string>
    </property>
   </widget>
   <widget class="QLabel" name="titulo">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>SO - Unidade 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="subtitulo">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>301</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trabalho dos trens - DeadLocks e Sem√°foros</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_vel_trem1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>460</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="sliderPosition">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_vel_trem2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>490</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="sliderPosition">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_vel_trem3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>520</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="sliderPosition">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_vel_trem4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>470</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_vel_trem5">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>500</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="titulo_velocidade">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>410</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>Velocidades</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_desc_trem1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>460</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: purple}</string>
    </property>
    <property name="text">
     <string>T1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_desc_trem2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>490</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: orange}</string>
    </property>
    <property name="text">
     <string>T2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_desc_trem3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>520</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: rgb(114, 159, 207)}</string>
    </property>
    <property name="text">
     <string>T3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_desc_trem4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>470</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: red}</string>
    </property>
    <property name="text">
     <string>T4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_desc_trem5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>500</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: rgb(138, 226, 52)}</string>
    </property>
    <property name="text">
     <string>T5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho_horizontal_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>110</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho_horizontal_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>230</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho_horizontal_6">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>230</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho_horizontal_7">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>350</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho_horizontal_8">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>350</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_vel_trem1">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>450</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vel_trem2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>480</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vel_trem3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>510</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vel_trem4">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>460</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vel_trem5">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>500</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <zorder>label_trilho_horizontal_1</zorder>
   <zorder>label_trilho_vertical_1</zorder>
   <zorder>label_trilho_horizontal_2</zorder>
   <zorder>label_trilho_horizontal_5</zorder>
   <zorder>label_trilho_vertical_2</zorder>
   <zorder>label_trilho_horizontal_6</zorder>
   <zorder>label_trilho_horizontal_8</zorder>
   <zorder>label_trilho_horizontal_4</zorder>
   <zorder>label_trilho_horizontal_7</zorder>
   <zorder>label_trilho_vertical_5</zorder>
   <zorder>label_trilho_horizontal_3</zorder>
   <zorder>label_trilho_vertical_4</zorder>
   <zorder>label_trilho_vertical_3</zorder>
   <zorder>label_trilho_vertical_6</zorder>
   <zorder>label_trilho_vertical_7</zorder>
   <zorder>label_trem5</zorder>
   <zorder>label_trem3</zorder>
   <zorder>titulo</zorder>
   <zorder>subtitulo</zorder>
   <zorder>slider_vel_trem1</zorder>
   <zorder>slider_vel_trem2</zorder>
   <zorder>slider_vel_trem3</zorder>
   <zorder>slider_vel_trem4</zorder>
   <zorder>slider_vel_trem5</zorder>
   <zorder>titulo_velocidade</zorder>
   <zorder>label_desc_trem1</zorder>
   <zorder>label_desc_trem2</zorder>
   <zorder>label_desc_trem3</zorder>
   <zorder>label_desc_trem4</zorder>
   <zorder>label_desc_trem5</zorder>
   <zorder>label_trem4</zorder>
   <zorder>label_trem1</zorder>
   <zorder>label_trem2</zorder>
   <zorder>label_vel_trem1</zorder>
   <zorder>label_vel_trem2</zorder>
   <zorder>label_vel_trem3</zorder>
   <zorder>label_vel_trem4</zorder>
   <zorder>label_vel_trem5</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>848</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
